docker/zap-api-scan.py

```diff
@@ -418,9 +419,14 @@ def main(argv):
                 logging.debug('Import OpenAPI URL ' + target_url)
                 res = zap.openapi.import_url(target, host_override)
                 urls = zap.core.urls()
+
                 if host_override:
-                    target = urljoin(target_url, '//' + host_override)
-                    logging.info('Using host override, new target: {0}'.format(target))
+                    if urlparse(host_override).scheme:
+                        target = host_override
+                    else:
+                        target = urljoin(target_url, '//' + host_override)
+                    logging.info(
+                        'Using host override, new target: {0}'.format(target))
```

Comment:
In this log entry maybe remove “host”, and just let it be override? 🤷‍♂️

This may not get a second set of eyes until Monday so don’t feel like you have to rush, it’s fine to wait for agreement or disagreement 😉

Reply:
@kingthorin Gotcha, I'll check again Monday/Tuesday. Thanks!

Reply:
I take the thumbs up to mean they agree 😁